js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadRoot.tsx

```diff
@@ -48,6 +48,44 @@ export const AssetLaunchpad = ({
   );
 };
 
+export const BackfillLaunchpad = ({
+  repoAddress,
+  sessionPresets,
+  assetJobName,
+  open,
+  setOpen,
+  onSaveConfig,
+}: {
+  repoAddress: RepoAddress;
+  sessionPresets?: Partial<IExecutionSession>;
+  assetJobName: string;
+  open: boolean;
+  setOpen: (open: boolean) => void;
+  onSaveConfig: (config: any) => void;
+}) => {
+  const title = 'Config Editor';
+
+  return (
+    <Dialog
+      style={{height: '90vh', width: '80%'}}
+      isOpen={open}
+      canEscapeKeyClose={false}
+      canOutsideClickClose={true}
```

Comment:
In general you want these either both enabled or both disabled. If it's important for the user to confirm submission (or cancel it) in some way, you should disable them.

So a decent rule of thumb is that if it's a read-only dialog, they can both be enabled since there's no work to lose accidentally on a rogue click (`true` is the default for these props). And if it's not read-only, you might want to set them to false to make sure the user has to commit to something.

Reply:
messed around with this and realized there's another interaction that's important here:

1. add config, save it
2. realize you want to add something else
3. open this again

added some code to wire the saved config back in if it's provided